webrtc.html

```diff
@@ -2363,11 +2363,14 @@
                 which point the <code><a>RTCPeerConnection</a></code> can fully
                 adopt the pending local description, or rollback to the current
                 description if the remote side rejected the change.</p>
+                <p>As noted in <span data-jsep="ice-candidate-pool">[[!JSEP]]</span>,
+                calling this method triggers the ICE candidate gathering process
```

Comment:
The normative requirement is in JSEP, so if anything this should be in a `<div class="note">`. We've been pretty lax about mixing normative and non-normative text...